.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,37 +1,29 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Create a bug report to help us improve the project.
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+Write a clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
+Steps to reproduce the bug:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
 **Expected behavior**
-A clear and concise description of what you expect to happen.
+Write a clear and concise description of what you expect to happen.
 
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Desktop (please complete the following information):**
- - OS: 
- - Browser 
- - Version [e.g. 22]
-
-**Smartphone (please complete the following information):**
- - Device
- - OS
- - Browser
- - Version [e.g. 22]
-
-Add some pictures too.
+**Device Information (please complete the following information):**
+ - Device: (e.g. Macbook Air M2)
+ - OS and Version:  (e.g. Windows 10)
+ - Browser: (e.g. Chrome)
```

Comment:
It would also be good to have:
- Video card: (e.g Nvidia GTX 970 with proprietary driver)
- Screen resolution and desktop scaling factor (eg 1080p, 200%)

Reply:
Okay, I'll add that too.